pyro/infer/mcmc/nuts.py

```diff
@@ -53,22 +53,25 @@ class NUTS(HMC):
         :mod:`torch.distributions.constraint_registry`.
 
     Example::
-
-        true_coefs = torch.tensor([1., 2., 3.])
-        data = torch.randn(2000, 3)
-        dim = 3
-        labels = dist.Bernoulli(logits=(true_coefs * data).sum(-1)).sample()
-
-        def model(data):
-            coefs_mean = torch.zeros(dim)
-            coefs = pyro.sample('beta', dist.Normal(coefs_mean, torch.ones(3)))
-            y = pyro.sample('y', dist.Bernoulli(logits=(coefs * data).sum(-1)), obs=labels)
-            return y
-
-        nuts_kernel = NUTS(model, adapt_step_size=True)
-        mcmc_run = MCMC(nuts_kernel, num_samples=500, warmup_steps=300).run(data)
-        posterior = EmpiricalMarginal(mcmc_run, 'beta')
-        print(posterior.mean)
+        >>> from pyro.infer import EmpiricalMarginal
+        >>> from pyro.infer.mcmc import MCMC
+
+        >>> true_coefs = torch.tensor([1., 2., 3.])
+        >>> data = torch.randn(2000, 3)
+        >>> dim = 3
+        >>> labels = dist.Bernoulli(logits=(true_coefs * data).sum(-1)).sample()
+
+        >>> def model(data):
+        ...     coefs_mean = torch.zeros(dim)
+        ...     coefs = pyro.sample('beta', dist.Normal(coefs_mean, torch.ones(3)))
+        ...     y = pyro.sample('y', dist.Bernoulli(logits=(coefs * data).sum(-1)), obs=labels)
+        ...     return y
+
+        >>> nuts_kernel = NUTS(model, adapt_step_size=True)
+        >>> mcmc_run = MCMC(nuts_kernel, num_samples=500, warmup_steps=300).run(data)
+        >>> posterior = EmpiricalMarginal(mcmc_run, 'beta')  # doctest: +SKIP
```

Comment:
This can be removed. I wanted to test if its possible to skip some steps from execution itself so that we can save time by skipping expensive steps, but doctest runs it even though it doesn't complain if any errors arise.